cookbook/form/data_transformers.rst

```diff
@@ -111,29 +111,100 @@ for converting to and from the issue number and the ``Issue`` object::
 Using the Transformer
 ---------------------
 
-Now that you have the transformer built, you just need to add it to your
-issue field in some form.
+As seen above our transformer requires an instance of an object manager. While for most
+use-cases using the default manager is fine we will let you pick the manager by it's name.
+In order to achieve this we will add a factory::
+
+    // src/Acme/TaskBundle/Form/DataTransformer/IssueToNumberTransformerFactory.php
+    namespace Acme\TaskBundle\Form\DataTransformer;
+
+    use Symfony\Bridge\Doctrine\ManagerRegistry;
```

Comment:
do you have any reason why you don't use `Doctrine\Common\Persistence\ManagerRegistry`, but require the specific bridge implementation?